src/main/java/core/basesyntax/figure/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figure;
+
+public class Square extends AbstractFigure implements Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: "
+                + getArea()
+                + " sq. units, side: "
+                + side
+                + " units, color: "
+                + color);
```

Comment:
You should use the 'name()' method instead of 'toString()' when you want to get the String representation of enum constants. If 'color' is an enum, then you should call 'color.name()'.